### PR TITLE
allow the user to use globs in the list of extra files

### PR DESCRIPTION
**Pull request by db48x**
and only issue a warning if any of them cannot be read, so that the rest of the documentation is still generated.

I am testing with https://github.com/CleverRaven/Cataclysm-DDA, if you’re curious.

### CODE REVIEW

File: CONFIG.md
Comment (rdmsr) [inline CONFIG.md:1]: The configuration file is named `cppdoc.toml`, not `config.toml`

**Review by rdmsr — CHANGES_REQUESTED** [review]
Thank you! The only problem I see with this is in the `CONFIG.md` file, the cppdoc configuration file is actually called `cppdoc.toml` not `config.toml`. Otherwise, this PR LGTM.

**Comment by rdmsr** [comment]
Do note that the pages you're testing on do not have markdown metadata, so getting the title of those pages will fail.
Also, be sure to test on header files and not source files.

File: CONFIG.md
Comment (db48x) [reply CONFIG.md:1]: bit of a typo there :)

**Pull request merged**